prowler/providers/aws/services/codebuild/codebuild_service.py

```diff
@@ -84,12 +84,21 @@ def _batch_get_projects(self, project):
             if project_info["source"]["type"] != "NO_SOURCE":
                 project.source = Source(
                     type=project_info["source"]["type"],
-                    location=project_info["source"]["location"],
+                    location=(
+                        project_info["source"]["location"]
+                        if project_info["source"].get("location") is not None
+                        else ""
+                    ),
                 )
             project.secondary_sources = []
             for secondary_source in project_info.get("secondarySources", []):
                 source_obj = Source(
-                    type=secondary_source["type"], location=secondary_source["location"]
+                    type=secondary_source["type"],
+                    location=(
+                        secondary_source["location"]
+                        if secondary_source.get("location") is not None
+                        else ""
```

Comment:
You can also do `secondary_source["location","") to have it in one line without the `if/else`